app/components/icon_list_item_component.html.erb

```diff
@@ -2,5 +2,5 @@
   <%= content_tag(:div, class: icon_css_class) do %>
     <%= render IconComponent.new(icon: icon) %>
   <% end %>
-  <div class="usa-icon-list__content"><%= content %></div>
+  <div class="usa-icon-list__content flex-2"><%= content %></div>
```

Comment:
Can you explain this change? This component is used in a number of screens, do we know what the potential impact this will have on those use-cases?

I'd wonder if we could just have a wrapper within the content we're rendering for the individual list item, i.e. inside the `with_item` in `app/views/sign_up/completions/show.html.erb` below.

Reply:
The parent li has display:flex on it and without adding flex-grow to the second one it wouldn't expand enough to have the change link hang to the right side of the div.
![noflexgrow](https://github.com/user-attachments/assets/982fd684-d709-44d9-891e-0d8203efd4db)
I couldn't find any other way of forcing the width on the second div to make room for that.
I did look at other places that use the icon list component and didn't see any issues. 
![Screenshot 2024-07-31 at 8 51 11 AM (2)](https://github.com/user-attachments/assets/e98bd2c1-b670-489e-aa92-2589e3b01ed3)

Reply:
After looking at this a bit closer, I think the more appropriate class to use here would be `grid-col-fill`, to get the grid column to fill the available space.

```suggestion
  <div class="usa-icon-list__content grid-col-fill"><%= content %></div>
```